test/test_optim_optim.py

```diff
@@ -16,7 +16,7 @@ def test_init(self):
         except BaseException:
             self.fail("__init__ failed.")
 
-        self.assertEquals(optimizer.max_grad_norm, 0)
+        self.assertEqual(optimizer.max_grad_norm, 0)
```

Comment:
why assertEqual without s?

Reply:
They are equivalent in python 2 but the plural version is not recommended with Python 3.0 actually, see [https://stackoverflow.com/questions/930995/assertequals-vs-assertequal-in-python](https://stackoverflow.com/questions/930995/assertequals-vs-assertequal-in-python)